firefox-ios/Client/Extensions/UIView+Multiwindow.swift

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import SnapKit
+
+extension UIView {
+    /// If the view is installed in a view hierarchy that is part of a window,
+    /// returns the window UUID. If not, returns nil.
+    var currentWindowUUID: WindowUUID? {
+        return (self.window as? BrowserWindow)?.uuid
+    }
```

Comment:
Note: this addition isn't necessary for this PR but is part of forthcoming work that we'll likely be using once themeing/Felt Privacy is updated for multi-window. If any concerns however LMK and I'm happy to remove and revisit later.